src/components/header/styleHeader/loginButton.scss

```diff
@@ -0,0 +1,17 @@
+button {
+    background: none;
+    color: rgb(241, 160, 8);
+    width: 120px;
+    height: 40px;
+    border: 2px solid rgb(241, 160, 8);
+    font-size: 15px;
+    border-radius: 4px;
+    transition: 0.6s;
+    overflow: hidden;
+    margin-right: 20px;
+
+    &:hover {
+        background: rgb(248, 72, 2);
```

Comment:
Define all the colors in one place with proper names and use them all around the app. It will be easy when you plan to restyle the app later.